### PR TITLE
Add FAQ Page

### PR DESCRIPTION
**Pull request by tzhelyazkova**
:question: :bookmark_tabs: 
To view the page do:
`npm run watch-faq` inside `cat17` folder
then access `http://localhost:8082/page/H%C3%A4ufige%20Fragen`

### CODE REVIEW

File: skins/cat17/package.json [hunk 1]
```diff
@@ -73,14 +73,17 @@
     "redux": "^3.7.2",
     "redux-promise": "^0.5.3",
     "underscore": "^1.9.1",
-    "underscore.path": "^0.1.4"
+    "underscore.path": "^0.1.4",
+    "vue": "^2.5.17"
```
Comment (RazShutyWMDE) [inline skins/cat17/package.json]: OMG! Yay!!!

File: skins/cat17/faq-page/.babelrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (gbirke) [inline skins/cat17/faq-page/.babelrc]: You can delete this file.

File: skins/cat17/faq-page/.eslintrc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (gbirke) [inline skins/cat17/faq-page/.eslintrc]: You can delete this file.

File: skins/cat17/faq-page/.gitignore [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.DS_Store
```
Comment (gbirke) [inline skins/cat17/faq-page/.gitignore]: Is this file still necessary? It probably was generated and everything in here should be covered by the parent .gitignore.

File: skins/cat17/faq-page/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# faq-page
```
Comment (gbirke) [inline skins/cat17/faq-page/README.md]: You can delete this file.

File: skins/cat17/faq-page/index.html [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (gbirke) [inline skins/cat17/faq-page/index.html]: You can delete this file.

File: skins/cat17/faq-page/karma.conf.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+// https://github.com/Nikku/karma-browserify
```
Comment (gbirke) [inline skins/cat17/faq-page/karma.conf.js]: You can delete this file.

File: skins/cat17/faq-page/src/Faq.vue [hunk 8]
```diff
@@ -0,0 +1,64 @@
+<template>
+ <div id="faq" class="container">
+ 	<h2>Häufige Fragen</h2>
```
Comment (gbirke) [inline skins/cat17/faq-page/src/Faq.vue]: Remember to make this translatable. If this makes the 5-point task too big or you don't consider this to be part of the ticket, please create a new ticket so we don't forget.

Comment (tzhelyazkova) [reply skins/cat17/faq-page/src/Faq.vue]: I thought this will be part of the task to load the actual content of the page :thinking:

**Comment by tzhelyazkova** [comment]
Please review and merge this #1338 as well, cause the new icon is needed in the FAQ design.

File: skins/cat17/package.json [hunk 9]
```diff
@@ -59,26 +74,33 @@
...
   },
   "browserify": {
+    "browser": {
+      "vue": "vue/dist/vue.common.js"
+    },
     "transform": [
       [
-        "polyify",
+        "vueify",
```
Comment (gbirke) [inline skins/cat17/package.json]: Ok, so in this setup we've dropped `polify`, which is ok because that was only for IE <10 support. This PR should then remove it from the dev packages in package.json and also drop the configuration object that follows this line.

File: skins/cat17/src/sass/layouts/pages/faq.scss [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+		background-color: white;
+		height: 40px;
+		border: 1px solid #ddd;
+		border-left: 0px;
+
+	}
+	input[type=text], button {
+		display: inline-block;
+	}
+	#answer {
```
Comment (gbirke) [inline skins/cat17/src/sass/layouts/pages/faq.scss]: Please don't use IDs, especially for items that appear multiple times on the page.

File: skins/cat17/.eslintrc [hunk 11]
```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "standard",
```
Comment (gbirke) [inline skins/cat17/.eslintrc]: This is the wrong coding standard, please use the wikimedia one.

File: skins/cat17/.eslintrc [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+  "plugins": [
+    "html"
+  ],
+  "env": {
+    "jasmine": true
```
Comment (gbirke) [inline skins/cat17/.eslintrc]: Until we use Jasmine as our testing framework, please remove this global.
We also might need to add `WMDE` here.

File: skins/cat17/package.json [hunk 13]
```diff
@@ -9,10 +9,13 @@
   },
   "scripts": {
     "watch": "gulp watch",
+    "watch-faq": "watchify -vd -p browserify-hmr -e faq-page/src/main.js -o ../../web/skins/cat17/scripts/faq.js",
+    "build-faq": "cross-env NODE_ENV=production browserify -g envify -p [ vueify/plugins/extract-css -o ../../web/skins/cat17/css/faq.css ] -e faq-page/src/main.js | uglifyjs -c warnings=false -m > ../../web/skins/cat17/scripts/faq.js",
+    "lint": "eslint --ext .js,.vue src",
```
Comment (gbirke) [inline skins/cat17/package.json]: This will produce thousands of errors. Maybe it's because of the wrong coding style, but I guess we also need to restrict this to the `faq-page` directory for now.

**Comment by gbirke** [comment]
I'm working on a solution for the build error ...
